components/home-page/bal-action/action-bbe/restful-api.md

```diff
@@ -17,11 +17,11 @@ table<Album> key(id) albums = table [
     ];
 
 service / on new http:Listener(port) {
-    resource function get albums() returns Album[] {
+    resource method get albums() returns Album[] {
```

Comment:
Should not be changed in the code snippets

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.